### ISSUE TITLE
jaws new fails without .aws/credentials

### ISSUE THREAD

**Issue by remicastaing** [jaws-stack/JAWS#72]
Without .aws/credentials, I get this:
```
REMI:test remi$ jaws new
       ____   _____  __      __  _________ 
      |    | /  _  \/  \    /  \/   _____/ 
      |    |/  /_\  \   \/\/   /\_____  \  
  /\__|    /    |    \        / /        \ 
  \________\____|__  /\__/\__/ /_________/ 

      *** The Server-less Framework ***     

? Please enter the ACCESS KEY ID for your ADMIN AWS IAM User: AZERTY
? Please enter the SECRET ACCESS KEY for your ADMIN AWS IAM User: +AZERTERTERTERDFVDFGDF
? Type a name for your new project (max 20 chars): test2
? Which stage would you like to create? (you can import more later) test
? Email would you like to use for AWS alarms: remi@test.com
Unhandled rejection TypeError: Cannot read property 'trim' of undefined
    at Object.module.exports.setAwsAdminCreds (/Users/remi/Workspaces/JAWS/JAWS/lib/utils/index.js:75:35)
    at Promise.then.adminEnv (/Users/remi/Workspaces/JAWS/JAWS/lib/commands/new.js:160:21)
    at tryCatcher (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._resolveFromResolver (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/promise.js:476:31)
    at new Promise (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/promise.js:69:37)
    at Object.JAWS.new (/Users/remi/Workspaces/JAWS/JAWS/lib/commands/new.js:137:12)
    at /Users/remi/Workspaces/JAWS/JAWS/bin/jaws:18:17
    at tryCatcher (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/promise.js:503:31)
    at Promise._settlePromiseAt (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/promise.js:577:18)
    at Promise._settlePromises (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/promise.js:693:14)
    at Async._drainQueue (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/remi/Workspaces/JAWS/JAWS/node_modules/bluebird/js/main/async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:358:17)
```

**Comment by doapp-ryanp** [comment]
This branch is not ready yet, please do not test.

**Comment by remicastaing** [comment]
I wasn't testing, I just ran into it! ;)
